### DISCUSSION TITLE
Delay exchangestart timeout

### DISCUSSION THREAD

**Discussion by dong9205** [rabbitmq/rabbitmq-delayed-message-exchange#191] [General]
Hello, michaelklishin 
I have two questions here.
1. The producer sends a message to the delay Exchange, which stores the message locally. After the X-delay time, the message is routed to the queue according to the route.I found that when messages were routed to the corresponding queue, the DCD file in the local file of the delay switch was still not released and still occupied a large amount of disk space, which indirectly led to my problem 2 situation.

2. Restart RabbitMQ when a large number of messages have accumulated on the delay switch.I see that Port 15672 of RabbitMQ_management is accessed normally.However, after one minute, I found that RabbitMQ stopped working. After checking the log, I found that I could only clear the messages for recovery later because of the startup timeout.

**Comment by michaelklishin** [comment]
The plugin relies on the currently used schema database and does not control how it manages its on-disk files.

We cannot comment on "stopped working". We don't guess on this team, so you have to provide at least some metrics and details on what exactly that means. If we cannot reproduce a problem, we cannot
fix it (or it would be guessing at best).
